db/seeds.rb

```diff
@@ -18,21 +18,33 @@
 Department.delete_all
```

Comment:
Running rails db:seed fails with errors because of the order of these two lines.  You want:
```
Employee.delete_all
Department.delete_all
```
 instead of the other way around.
 
 This is because of the foreign key dependency that Employees have on Departments.
 
 Other than that, looks great!